src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -49,19 +52,9 @@ export const monoPlatform: Platform = {
     return new Promise<void>((resolve, reject) => {
       attachDebuggerHotkey(resourceLoader);
 
-      // dotnet.js assumes the existence of this
-      window['Browser'] = {
-        // eslint-disable-next-line @typescript-eslint/no-empty-function
-        init: () => { },
-      };
-
-      // Emscripten works by expecting the module config to be a global
-      // For compatibility with macOS Catalina, we have to assign a temporary value to window.Module
-      // before we start loading the WebAssembly files
-      addGlobalModuleScriptTagsToDocument(() => {
-        window['Module'] = createEmscriptenModuleInstance(resourceLoader, resolve, reject);
-        addScriptTagsToDocument(resourceLoader);
-      });
+      createEmscriptenModuleInstance(resourceLoader, resolve, reject);
+    }).then(() => {
+      window['Module'] = Module;
```

Comment:
Is this necessary? Doesn't `disableDotnet6Compatibility: false` do it anyway?

Reply:
You are right